From 2bfbce8a3cea0b7548adb94be146406138d7c251 Mon Sep 17 00:00:00 2001
From: Bariccatti <bariccatti@gmail.com>
Date: Sat, 25 Apr 2026 16:18:04 -0300
Subject: [PATCH] Remove unused failToLoadFromDriverSuffixProperty variable

Patch 0010 removed the usage of failToLoadFromDriverSuffixProperty
but left the declaration and assignment, causing -Werror=unused-but-set-variable.
---
 opengl/libs/EGL/Loader.cpp | 2 --
 1 file changed, 2 deletions(-)

diff --git a/opengl/libs/EGL/Loader.cpp b/opengl/libs/EGL/Loader.cpp
index ad7946c..1d25391 100644
--- a/opengl/libs/EGL/Loader.cpp
+++ b/opengl/libs/EGL/Loader.cpp
@@ -289,7 +289,6 @@ void* Loader::open(egl_connection_t* cnx) {
     }
 
     // Finally, try to load default driver.
-    bool failToLoadFromDriverSuffixProperty = false;
     if (!hnd) {
         // Start by searching for the library name appended by the system
         // properties of the GLES userspace driver in both locations.
@@ -304,7 +303,6 @@ void* Loader::open(egl_connection_t* cnx) {
             hnd = attempt_to_load_system_driver(cnx, prop.c_str(), true);
             if (!hnd) {
                 ALOGD("Failed to load drivers from property %s with value %s", key, prop.c_str());
-                failToLoadFromDriverSuffixProperty = true;
             }
 
             // Abort regardless of whether subsequent properties are set, the value must be set
-- 
2.53.0.windows.3

